### PR TITLE
Additional phenotype codes to support AnVIL data

### PR DESCRIPTION
**Pull request by bwalsh**
This PR contains additional phenotype codes to support anvil ResearchStudy.condition and Observation.phenotype

## ETL overview

image

### Implementation

[Terra Notebook](https://github.com/anvilproject/client-apis/blob/fhir/pyAnVIL/notebooks/notebooks_dashboard-fhir-0.0.5rc5.ipynb)

### Extracted FHIR source data

image

### Results of local docker testing

image

### CODE REVIEW

File: site_root/input/resources/terminology/CodeSystem-doid.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    "id": "doid",
+    "url": "http://purl.obolibrary.org/obo/doid.owl",
+    "name": "doid",
+    "title": "DOID",
+    "status": "draft",
+    "experimental": false,
+    "description": "DOID",
+    "caseSensitive": true,
+    "content": "complete",
+    "count": 1,
```
Comment (liberaliscomputing) [inline site_root/input/resources/terminology/CodeSystem-doid.json:12]: `"count"` should be equal to the number of total concepts in the code system.

File: site_root/input/resources/terminology/CodeSystem-doid.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    "resourceType": "CodeSystem",
+    "id": "doid",
+    "url": "http://purl.obolibrary.org/obo/doid.owl",
+    "name": "doid",
+    "title": "DOID",
+    "status": "draft",
+    "experimental": false,
+    "description": "DOID",
+    "caseSensitive": true,
+    "content": "complete",
```
Comment (liberaliscomputing) [inline site_root/input/resources/terminology/CodeSystem-doid.json:11]: Please refer to https://www.hl7.org/fhir/valueset-codesystem-content-mode.html.

File: site_root/input/resources/terminology/CodeSystem-hpo.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "resourceType": "CodeSystem",
+    "id": "hpo",
+    "url": "http://purl.obolibrary.org/obo/hp.owl",
+    "name": "hpo",
+    "title": "HPO",
+    "status": "draft",
+    "experimental": false,
+    "description": "HPO",
+    "caseSensitive": true,
+    "content": "complete",
```
Comment (liberaliscomputing) [inline site_root/input/resources/terminology/CodeSystem-hpo.json:11]: As commented above.

File: site_root/input/resources/terminology/CodeSystem-snomedct.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  "resourceType": "CodeSystem",
+  "id": "snomedct",
+  "url": "http://snomed.info/sct",
+  "name": "snomed-ct",
+  "title": "SNOMED CT",
+  "status": "draft",
+  "experimental": false,
+  "description": "SNOMED CT",
+  "caseSensitive": true,
+  "content": "complete",
```
Comment (liberaliscomputing) [inline site_root/input/resources/terminology/CodeSystem-snomedct.json:11]: As commented above.

File: site_root/input/resources/terminology/CodeSystem-snomedct.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  "count": 2,
```
Comment (liberaliscomputing) [inline site_root/input/resources/terminology/CodeSystem-snomedct.json:12]: As commented above.

File: site_root/input/resources/terminology/ValueSet-phenotype-codes.json [hunk 6]
```diff
@@ -13,6 +13,12 @@
     "include": [
```
Comment (liberaliscomputing) [inline site_root/input/resources/terminology/ValueSet-phenotype-codes.json:13]: Don't we include SNOMED CT as well?

File: site_root/input/resources/terminology/CodeSystem-doid.json (hunk 2 above)
Comment (bwalsh) [reply site_root/input/resources/terminology/CodeSystem-doid.json:11]: Thanks.  Looking at the possibilities, I'd assume `fragment` is the correct value to use in this case?

File: site_root/input/resources/terminology/ValueSet-phenotype-codes.json (hunk 6 above)
Comment (bwalsh) [reply site_root/input/resources/terminology/ValueSet-phenotype-codes.json:13]: @liberaliscomputing Are snomed just used for  "phenotype-observation-codes" ?

Comment (liberaliscomputing) [reply site_root/input/resources/terminology/ValueSet-phenotype-codes.json:13]: Ah yes. We don't need to include SNOMED CT here.

**Comment by liberaliscomputing** [comment]
@bwalsh, I am holding off approving this PR since I will meet with the Smile CDR's developers tomorrow to talk about the remote terminology service. If the issue is resolved, we would want to remove all the custom CodeSystems from our repo. In the meantime, could you check if the code systems you are adding are here: https://r4.ontoserver.csiro.au/fhir/CodeSystem/

**Review by liberaliscomputing — APPROVED** [review]
Looks good to get merged in!

**Review by torstees — APPROVED** [review]
This looks straight forward and the issues Meen pointed out have been fixed. So, it certainly appears ready to go to me.

**Pull request merged**